### PR TITLE
centos box

### PR DESCRIPTION
**Pull request by wallies**

### CODE REVIEW

File: packer/centos-7.0_x64.json [hunk 1]
```diff
@@ -0,0 +1,83 @@
+{
+  "variables": {
+    "ssh_name": "vagrant",
+    "ssh_pass": "vagrant",
+    "hostname": "ubuntu",
```
Comment (tayzlor) [inline packer/centos-7.0_x64.json]: hostname ubuntu?

File: packer/centos-7.0_x64.json [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    "consul_version": "0.5.0",
+    "weave_version": "latest_release",
+    "build_version": "{{ timestamp }}"
+  },
+  "builders": [
+    {
+      "type": "virtualbox-iso",
+      "guest_os_type": "RedHat_64",
+      "headless": true,
+      "vm_name": "mesos-{{user `mesos_version`}}_centos-7.0_x64_virtualbox_{{user `build_version`}}",
```
Comment (tayzlor) [inline packer/centos-7.0_x64.json]: should be apollo-mesos (probaly leave out the mesos_version too)

File: packer/scripts/centos/install_dnsmasq.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+set -eux
+set -o pipefail
+
+yum install -y dnsmasq
+chkconfig dnsmasq on
+
+sudo systemctl stop dnsmasq.service
+sudo systemctl disable dnsmasq.service
+echo "server=/consul/127.0.0.1#8600" > /etc/dnsmasq.d/10-consul
```
Comment (tayzlor) [inline packer/scripts/centos/install_dnsmasq.sh]: remove this - it gets provisioned via ansible

File: packer/scripts/centos/systemd/consul.conf [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+# Make sure to use all our CPUs, because Consul can block a scheduler thread
+export GOMAXPROCS=`nproc`
+
+# Get the public IP
+BIND=`ifconfig eth0 | grep "inet addr" | awk '{ print substr($2,6) }'`
+
+ExecStart=/usr/bin/consul agent \
+  -data-dir /mnt/consul \
+  -config-dir /etc/consul.d \
+  -bind=$BIND \
```
Comment (tayzlor) [inline packer/scripts/centos/systemd/consul.conf]: remove -bind (it can be specified in the consul.json file)

**Comment by wallies** [comment]
im gonna reject the centos box PR, but leave the branch there, for when we have moved everything we can into containers, or at least #129 & #105. Will make things alot easier

**Pull request closed without merging**